omou-frontend/src/App.test.js

```diff
@@ -1,9 +1,78 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+const puppeteer = require('puppeteer');
+const faker = require('faker');
+
+const person = {
+  name: faker.name.firstName() + ' ' + faker.name.lastName(),
+  email: "calvin@email.com",
+  phone: faker.phone.phoneNumber(),
+  message: faker.random.words(),
+  password: "password"
+};
+
+
+const isDebugging = () => {
+  let debugging_mode = {
+    headless: false,
+    slowMo: 50,
+    devtools: true,
+  };
+  return process.env.NODE_ENV === 'debug' ? debugging_mode : {};
+};
+
+let browser;
+let page;
+beforeAll(async () => {
+  browser = await puppeteer.launch(isDebugging());
+  page = await browser.newPage();
+  await page.goto('http://localhost:3000/');
```

Comment:
Use the constant `appUrlBase` instead of hardcoding the URL, so it can be easily changed later if need be.